### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by ahmedbahgattt** [Homebrew/discussions#3479] [Casks]
I have MacBook m1  and my terminal  start with that 

Last login: Sun Jul 31 02:23:22 on ttys000
/Users/ahmedbahgattt/.zshrc:export:1: not valid in this context: Fusion.app/Contents/Public:/opt/homebrew/bin:/opt/homebrew/bin

how can I fix it

**Comment by gromgit** [comment]
What's the output of `grep /opt/homebrew ~/.zshrc`?

**Reply by ahmedbahgattt** [reply]
export PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/opt/homebrew/bin:/opt/homebrew/bin
export PATH="/opt/homebrew/bin:$PATH"

**Reply by carlocab** [reply]
The space in `VMWare Fusion.app` is messing up your `.zshrc`. Either quote or escape appropriately. In any case, this is not an issue with Homebrew.
